benchmarks/benchmarks/bench_random.py

```diff
@@ -34,11 +34,11 @@ class Randint(Benchmark):
 
     def time_randint_fast(self):
         """Compare to uint32 below"""
-        np.random.randint(0, 2**30, size=10**5)
+        np.random.randint(low=0, high=2**30, size=10**5)
```

Comment:
The API would become dated if this PR is merged because `low` no longer is a positional argument.  While it may be worth adding <b>a test</b> to ensure the non-keyword argument call still works, I don't think it should persist in the code-base with this change.